deck-o-cards.js

```diff
@@ -1,33 +1,37 @@
 // 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
 // Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
-function deck_o_cards() {
+ function deck_o_cards() {
   var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
   var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
 
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
-
+for(i=0;i<values.length;i++) {
+    var value = values[i];
+  for(j=0;j<suits.length;j++){
+        var suit = suits[j];
+      cards = ["suit: " + suit + ", " + "value: " + value];
```

Comment:
You're getting close here, but what you want is a card _object_ not an array. i.e. `card={ }`. Then inside that object we want two properties with keys and values. The keys are `suit` and `value` while their values are found using the index for whatever loop you're in. 

```js
for(i=0;i<values.length;i++) {
     // create an empty card object each time you loop through the values array
   var card = {};
   for(j=0;j<suits.length;j++){
         // then set the properties inside that card object when you do the second loop through the suits
         card = {
             value: values[i],
             suit: suits[j]
        }
        // now push that new card object into the cards array (created on line 7)
        cards.push(card);
    }
}
```